runtests.py

```diff
@@ -74,40 +74,44 @@
 def main(argv):
     parser = ArgumentParser(usage=__doc__.lstrip())
     parser.add_argument("--verbose", "-v", action="count", default=1,
-                        help="more verbosity")
+                        help="Add one verbosity level to pytest. Default is 0")
     parser.add_argument("--debug-info", action="store_true",
-                        help=("add --verbose-cfg to build_src to show compiler "
-                              "configuration output while creating "
+                        help=("Add --verbose-cfg to build_src to show compiler"
+                              " configuration output while creating "
```

Comment:
By convention we put the space on the end of the previous line, as things were before you changed this line

Reply:
Does this convenience override E501 (line length), or should it fulfill both (so I should move compiler to the next line)?

Reply:
Just break the line into shorter segments.